canary_tests.sh

```diff
@@ -11,31 +12,17 @@ fi
 # See https://stackoverflow.com/a/1482133 for an explanation of this next line. It gets the
 # directory in which this script resides.
 this_dir="$(dirname -- "$( readlink -f -- "$0"; )")";
-echo "this_dir is $this_dir"
 pushd "$this_dir" > /dev/null
 
 # This script is adapted from
 # https://viam.atlassian.net/wiki/spaces/ENG/pages/155451406/Data+ML+Canary+Testing+Bot
 
-now=$(date +"%T")
-echo "Current time : $now"
-# stop the viam-server first
-echo "stopping viam-server"
```

Comment:
Making the justification more obvious than what's in a marked-as-resolved comment on this PR: The `set -x` at the top of the file has us print out every command that runs. We don't need to echo/print what we're going to do, because the very next line of output is going to be `systemctl stop viam-canary`.